lms/djangoapps/courseware/access.py

```diff
@@ -173,53 +182,6 @@ def has_access(user, action, obj, course_key=None):
                     .format(type(obj)))
 
 
-# ================ Implementation helpers ================================
-
-def has_staff_access_to_preview_mode(user, obj, course_key=None):
```

Comment:
So technically if i'm previewing a course as learner then i should see what learner will be seeing on courses. i.e. learners won't be able to see draft versions. @sstack22 Correct me if i'm wrong.